views/publications-details.html

```diff
@@ -0,0 +1,40 @@
+{% extends "layouts/two-columns.html" %}
+
+{% block main %}
+<article>
+  This publication was last updated on
+  <time datetime={{record.updated_at}}>{{ publication.updated_at | formatDate }}</time>
```

Comment:
Actually, just remove this last updated time here. I don't think we keep updating publications as we do with sources, so there's no need for it.